src/brasil/gov/tiles/tiles/groupcarousel.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+from brasil.gov.tiles import _
+from collective.cover.interfaces import ITileEditForm
+from collective.cover.tiles.carousel import CarouselTile
+from collective.cover.tiles.list import IListTile
+from collective.cover.widgets.textlinessortable import TextLinesSortableFieldWidget
+from plone.autoform import directives as form
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from zope import schema
+from zope.interface import implementer
+
+
+class IGroupCarouselTile(IListTile):
```

Comment:
we're inheriting here from `IListTile` but below we inherit from `CarouselTile`; why we're doing this? that's confusing and error prone.

Reply:
inheriting from CarouselTile to reuse the code, not to repeat the code [1]
inheriting from CarouselTile to reuse the code, not to repeat the code [2]

[1] https://github.com/collective/collective.cover/blob/master/src/collective/cover/tiles/list.py#L30-L91
[2] https://github.com/collective/collective.cover/blob/master/src/collective/cover/tiles/carousel.py#L36-L130